docs/building-apps/wallet/sep6.mdx

```diff
@@ -0,0 +1,272 @@
+
+Let's add some KYC info for our account using sample data. Binary data (eg. image data), needs to be sent in a separate field. The fields allowed to send to the anchor are described in [SEP-9].
+
+<CodeExample>
+
+```typescript
+const sep12 = await anchor.sep12(authToken);
+
+await sep12.add({
+  sep9Info: {
+    first_name: "john",
+    last_name: "smith",
+    email_address: "123@gmail.com",
+    bank_number: "12345",
+    bank_account_number: "12345",
+  },
+  sep9BinaryInfo: {
+    photo_id_front: Buffer.from("./path/to/image/front"),
+    photo_id_back: Buffer.from("./path/to/image/back"),
+  },
+});
+```
+
+</CodeExample>
+
+## Start a withdrawal
+
+Starting a withdrawal is similar to deposit, and has the same response types as described earlier.
+
+<CodeExample>
+
+```typescript
+const resp = await sep6.withdraw({
+  authToken,
+  params: {
+    asset_code: "SRT",
+    account: accountKp.publicKey,
+    type: "bank_account",
+    dest: "123",
+    dest_extra: "12345",
+  },
+});
+```
+
+</CodeExample>
+
+## Get exchange info
+
+If the anchor supports [SEP-38] quotes, it can support deposits that make a bridge between non-equivalent assets. For example, an anchor recieves BRL via bank transfer and in return sends USDC (of equivalent value minus fees) to the user on Stellar.
+
+The sep6 exchange functions allow a user to start an exchange deposit or withdrawal with the anchor, and the anchor can communicate back next steps to the user.
+
+First, let's start a deposit exchange with sample data. Assets are described using the [SEP-38] scheme.
+
+<CodeExample>
+
+```typescript
+const resp = await sep6.depositExchange({
+  authToken,
+  params: {
+    destination_asset:
+      "stellar:SRT:GCDNJUBQSX7AJWLJACMJ7I4BC3Z47BQUTMHEICZLE6MU4KQBRYG5JY6B",
+    source_asset: "iso4217:USD",
+    amount: "10",
+  },
+});
+```
+
+</CodeExample>
+
+The response follows the same types as all the deposits and withdrawals for SEP-6.
+
+Now let's create a withdrawal exchange, which follows the same format as the deposit exchange. We also specify it's bank account withdrawal using the `type` field.
+
+<CodeExample>
+
+```typescript
+const resp = await sep6.withdrawExchange({
+  authToken,
+  params: {
+    destination_asset: "iso4217:USD",
+    source_asset:
+      "stellar:SRT:GCDNJUBQSX7AJWLJACMJ7I4BC3Z47BQUTMHEICZLE6MU4KQBRYG5JY6B",
+    amount: "10",
+    type: "bank_account",
+  },
+});
+```
+
+</CodeExample>
+
+The response follows the same types as all the deposits and withdrawals for SEP-6.
+
+## Getting Transaction Info
+
+On the typical flow, the wallet would get transaction data to notify users about status updates. This is done via the [SEP-6] `GET /transaction` and `GET /transactions` endpoint.
+
+### Tracking Transaction
+
+Let's look into how to use the sdk to track transaction status changes. We will use the `Watcher` class for this purpose. First, let's initialize it and start tracking a transaction.
```

Comment:
For sep-24 watcher also has refresh, is it not the case for SEP-24?

Reply:
good catch 👍 https://github.com/stellar/stellar-docs/pull/283/commits/70257245562bcba23ed19eef415cf106495077f3